src/routes/+page.svelte

```diff
@@ -54,17 +62,18 @@
 
 	async function fetchGasEstimationFromGasNet(
 		chain: string
-	): Promise<{ estimationData: EstimationData; signature: string } | null> {
+	): Promise<{ paramsPayload: PayloadValues ,raw: string } | null> {
 		isLoading = true
 		transactionHash = null
 
 		try {
 			const { ethers } = await loadEthers()
 			const rpcProvider = new ethers.JsonRpcProvider(gasNetwork.url)
 			const gasNetContract = new ethers.Contract(gasNetwork.contract, gasnet.abi, rpcProvider)
-			const [estimation, signature] = await gasNetContract.getEstimation(chain)
-			transactionSignature = signature
-			return { estimationData: createEstimationObject(estimation), signature }
+			const payload = await gasNetContract.getValues(2,chain) 
```

Comment:
@lukanus what is `2` here? I am planning to move it into constants with a descriptive name